src/pages/products/products.js

```diff
@@ -5,11 +5,24 @@ import { Sidebar } from '../../components/sidebar/sidebar'
 import { useFilter } from '../../filter-context/filter-context'
 import { highToLow } from '../../filter-functions/high-to-low'
 import {lowToHigh} from '../../filter-functions/low-to-high'
+import { inStock } from '../../filter-functions/in-stock'
+import { fastDelivery } from '../../filter-functions/fast-delivery'
+import { priceRange } from '../../filter-functions/price-range'
+import { rating } from '../../filter-functions/rating'
+import { categoryHandler } from '../../filter-functions/category'
```

Comment:
you can use all these by putting them into a single file & then importing them

Reply:
ok got it.